integration/spark/src/main/scala/org/apache/spark/sql/execution/command/table/CarbonDropTableCommand.scala

```diff
@@ -117,6 +115,8 @@ case class CarbonDropTableCommand(
         viewDropCommands.foreach(_.processMetadata(sparkSession))
       }
 
+      CarbonEnv.getInstance(sparkSession).carbonMetaStore.dropTable(identifier)(sparkSession)
```

Comment:
add a comment here, whatever has been added in the PR description, so that later someone shouldn't change it back.